internal/transport/transport_test.go

```diff
@@ -2018,7 +2020,7 @@ func (s) TestClientDecodeHeaderStatusErr(t *testing.T) {
 			},
 			wantStatus: status.New(
 				codes.Internal,
-				": HTTP status code 0; transport: received the unexpected content-type \"application/json\"",
+				"transport: received the unexpected content-type \"application/json\"",
```

Comment:
Here there was no HTTP status in the input so it should be mentioned in the error.

Reply:
Done